backend/tests/legacy/test_api_snapshot.py

```diff
@@ -70,56 +51,6 @@ def test_pending_snapshot(
     print(f"bob_budget: {bob_budget} while alice_budget: {alice_budget} ")
 
 
-@pytest.mark.api
-def test_allocations(
-    client: Client, deployer: UserAccount, ua_alice: UserAccount, ua_bob: UserAccount
-):
-    alice_proposals = get_proposals_addresses(1)[:3]
-
-    # fund Octant
-    deployer.fund_octant(
-        address=client.config["WITHDRAWALS_TARGET_CONTRACT_ADDRESS"], value=400
-    )
-
-    # lock GLM from two accounts
-    deployer.transfer(ua_alice, 10000)
-    ua_alice.lock(10000)
-    deployer.transfer(ua_bob, 15000)
-    ua_bob.lock(15000)
-
-    # forward time to the beginning of the epoch 2
-    move_to_next_epoch(2)
-
-    # wait for indexer to catch up
-    epoch_no = wait_for_sync(client, 2)
-    print(f"indexed epoch: {epoch_no}")
-
-    # make a snapshot
-    res = client.pending_snapshot()
-    assert res["epoch"] > 0
-
-    ua_alice.allocate(1000, alice_proposals)
-    ua_bob.allocate(1000, alice_proposals[:1])
-
-    allocations = client.get_epoch_allocations(1)
-    unique_donors = set()
-    unique_proposals = set()
-
-    assert len(allocations["allocations"]) == 4
-    for allocation in allocations["allocations"]:
-        for key, val in allocation.items():
-            if key == "donor":
-                unique_donors.add(val)
-            if key == "amount":
-                assert int(val) > 0
-            if key == "proposal":
-                unique_proposals.add(val)
-            print("allocation items: ", allocation.items)
-
-    assert len(unique_donors) == 2
-    assert len(unique_proposals) == 3
-
-
```

Comment:
I have removed this as I am doing refactor and I would like to avoid future conflicts.
Allocation tests are added in:
https://github.com/golemfoundation/octant/pull/201